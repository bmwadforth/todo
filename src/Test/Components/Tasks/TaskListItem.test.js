import React from 'react';
import {shallow, configure} from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import configureStore from "redux-mock-store";
import {Provider} from "react-redux";
import TaskListItem from "../../../Components/Tasks/TaskListItem";
configure({adapter: new Adapter()});

describe("TaskListItem Component", () => {
    const initialState = {Tasks: {data: {}}};
    const mockStore = configureStore();
    let store;

    beforeEach(() => {
        store = mockStore(initialState);
    });

    it("Should Match Snapshot", () => {
        const wrapper = shallow(<Provider store={store}><TaskListItem id="1" title="Test" created={new Date()} /></Provider>);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
