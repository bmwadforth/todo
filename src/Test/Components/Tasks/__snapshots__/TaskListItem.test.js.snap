// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskListItem Component Should Match Snapshot 1`] = `"<div class=\\"task-row\\"><div class=\\"task-row-heading false undefined\\"><div class=\\"task-row-title\\"><p>Test</p><div class=\\"icon\\"><span><svg>caret.svg</svg></span></div></div></div></div>"`;
